app/services/articles/page_view_updater.rb

```diff
@@ -0,0 +1,33 @@
+module Articles
+  # This module is responsible for updating a specific page view for a given article and user.
+  #
+  # @see Articles::UpdatePageViewsWorker for the sibling that's responsible for recording page
+  #      views.
+  module PageViewUpdater
+    # @param article_id [Integer]
+    # @param user_id [Integer]
+    #
+    # @return [TrueClass] if we updated or created a PageView
+    # @return [FalseClass] if we did not update a PageView
+    #
+    # @note Regardless of return status, consider the business logic successful unless we raise an
+    #       exception.  The return value is present for easing testing.  The `find_or_create_by`
```

Comment:
```suggestion
    #       exception. The return value is present for easing testing. The `find_or_create_by`
```

Reply:
For sentences I always use two spaces between periods.  This is because my text editor allows for navigation by sentence.  So my strong preference is to keep two spaces.

Other use cases is that I can quickly expand my highlighted context, starting from cursor, to word, to sentence, to paragraph, to buffer.

Reply:
While I have no problems with two spaces after full stops (I'm old enough to remember that this used to be standard at one point) I'm not super convinced by this argument:

> This is because my text editor allows for navigation by sentence. So my strong preference is to keep two spaces.

This is a codebase shared by many people, IMHO we can't optimize for everyone's preferred tool choice.